### PR TITLE
Theme and project structure - updated

### PR DESCRIPTION
**Pull request by codyde**
This is updated using your new APIs - removed some of my examples since you cover them in this now. 

- Added some basic styling - just to clearly articulate "where" evaluations where happening and provide some minor education
- Created components for both client and server side explicitly so a user who viewed it would understand quickly what these components were doing
- Added cookies-next to handle storing the context as a cookie to show the example
- Added a .env.local thats got placeholder data so a user will know what to update.
- Removed your prettierrc config that normal users might not be setup for
- Updated the readme

### CODE REVIEW

File: app/invoice/page.tsx
Comment (yusinto) [inline app/invoice/page.tsx:1]: We can safely delete this file, because it's just a duplicate server comnponent example.

File: app/page.tsx [hunk 1]
```diff
@@ -1,27 +1,30 @@
-import App from '@/app/app';
-import LDButton from '@/app/LDButton';
-import { getLDContext } from '@/app/utils';
-import { useLDClientRsc } from '@/ld/server';
-import Link from 'next/link';
+import HelloClient from './components/helloLDClient';
+import HelloRSC from './components/helloLDRSC';
 
-// Server Components must useLDClientRsc for evaluation.
+// GOTCHA: page components must be async. Otherwise the ssr cache may not be initialized correctly.
```
Comment (yusinto) [inline app/page.tsx:4]: This comment is no longer correct. Page components can be async or not async. The only requirements are:

* Server Components must use `useLDClientRsc`
* Client Components must use `useLDClient`

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -104,14 +107,18 @@ export default async function RootLayout({
 import { getLDContext } from '@/app/utils';
 import { useLDClientRsc } from '@/ld/server';
 
-export default async function Page() {
+export default async function HelloRSC() {
```
Comment (yusinto) [inline README.md:110]:
```suggestion
export default async function HelloRsc() {
```

File: app/components/helloLDRSC.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { getLDContext } from '@/app/utils';
+import { useLDClientRsc } from '@/ld/server';
+
+export default async function HelloRSC() {
```
Comment (yusinto) [inline app/components/helloLDRSC.tsx:4]:
```suggestion
export default async function HelloRsc() {
```

File: app/components/helloLDRSC.tsx
Comment (yusinto) [inline app/components/helloLDRSC.tsx:1]: I have an ocd with too many uppercased acronyms. May I kindly suggest we rename these components `helloServerComponent` and `helloClientComponent`?

**Review by yusinto — APPROVED** [review]
A few minor comments otherwise lgtm. Thank you @codyde !

**Review by yusinto — APPROVED** [review]

**Comment by codyde** [comment]
All fixes are in! Merging!

**Pull request merged**